x-pack/plugins/saved_objects_tagging/public/components/edition_modal/create_or_edit_modal.tsx

```diff
@@ -58,10 +58,16 @@ export const CreateOrEditModal: FC<CreateOrEditModalProps> = ({
   const ifMounted = useIfMounted();
   const [submitting, setSubmitting] = useState<boolean>(false);
 
-  // we don't want this value to change when the user edit the name.
+  // we don't want this value to change when the user edits the tag
   // eslint-disable-next-line react-hooks/exhaustive-deps
-  const initialName = useMemo(() => tag.name, []);
-
+  const initialTag = useMemo(() => tag, []);
+  const tagHasBeenModified = useMemo(
```

Comment:
Seemed a shame to import lodash isEqual here to compare 3 properties of an object so kept it simple